docs/rsu.md

```diff
@@ -0,0 +1,46 @@
+# Rolling schema upgrade
+
+RSU, or Rolling Schema Upgrade, is a method for altering the structure of a database schema without bringing down the entire cluster. RSU updates one database at a time. This method means there's less downtime for your application. However, it also means there could be periods where different databases have different structures, which can sometimes cause problems.
+
+Thoroughly test RSU procedures in a non-production environment before deploying in production. Closely monitor the cluster during the upgrade to identify and promptly address any issues.
+
+Use RSU in the following scenarios:
+
+* You must perform the schema upgrade with as little disruption to the database service as possible. This is where RSU (Rolling Schema Upgrade) excels. By updating one node at a time, RSU reduces the overall downtime compared to traditional methods that require taking the entire cluster offline.
+
+* If the schema change is considered compatible with different schema versions if it can be applied to a database with an older schema without causing errors or data loss. This means that even though some databases in the cluster might have an older structure, the new schema change can still be applied to them successfully.
```

Comment:
If the schema change is considered compatible with different schema `versions, it` can be applied to a database with an older schema without causing errors or data loss.